src/com/palmergames/bukkit/towny/object/Town.java

```diff
@@ -1488,7 +1489,17 @@ public Set<Resident> getTrustedResidents() {
 	}
 	
 	public boolean hasTrustedResident(Resident resident) {
-		return trustedResidents.contains(resident);
+		Boolean isTrusted = false;
+		Town residentsTown = resident.getTownOrNull();
+		if (residentsTown != null) {
+			if (this.hasTrusted(residentsTown)) {
+				isTrusted = true;
+			}
+		}
+		if (!isTrusted) {
+			isTrusted = trustedResidents.contains(resident);
+		}
+		return isTrusted;
```

Comment:
```suggestion
		Town residentsTown = resident.getTownOrNull();
		return trustedResidents.contains(resident) || (residentsTown != null && this.hasTrusted(residentsTown));
```

hasTrusted() will have to change to hasTrustedTown()

Reply:
Implemented as requested.